src/notation/internal/notationinteraction.cpp

```diff
@@ -33,12 +33,14 @@
 #include <QDrag>
 #include <QRegularExpression>
 #include <QRegularExpressionMatch>
+#include <QBitmap>
```

Comment:
If the simplification I suggested below is indeed possible, don't forget to remove this #include or replace it with `#include <QPixmap>` :)